content/tutorials/pulumi-deployments-click-to-deploy/index.md

```diff
@@ -0,0 +1,120 @@
+you'll_learn:
+  - How to use the New Project Wizard to create and deploy a Pulumi project from your browser
+  - How to install and configure the Pulumi GitHub App
+  - How to trigger and manage deployments directly from the Pulumi Cloud console using click-to-deploy
+prereqs:
+    - Completion of the [Getting Started](/docs/get-started/) guide or familiarity with the basics of the Pulumi workflow.
+    - The [Pulumi CLI](/docs/install/)
+    - A [Pulumi Cloud account](https://app.pulumi.com/signup)
+    - An [Amazon Web Services](https://aws.amazon.com/) account
+    - A [GitHub account](https://github.com/) with admin rights to a Git repository or organization
+    - Familiarity with JavaScript, TypeScript, or Python
+estimated_time: 10
+allow_long_title: true
+---
+
+## What is Pulumi Deployments
+
+[Pulumi Deployments](/docs/pulumi-cloud/deployments/) is a fully managed service designed to automate cloud infrastructure. Deployments provide the building blocks designed for scaling your cloud infrastructure, with tools to power public and private cloud platforms, APIs, and self-service infrastructure portals. These options provide flexibility to incorporate Pulumi Deployments into your existing workflows.
+
+Deployments can be triggered through multiple methods: using the [REST API](/docs/pulumi-cloud/deployments/reference/#deployments-rest-api), integrating with GitHub via [push-to-deploy](/docs/pulumi-cloud/deployments/reference/#github-app-installation), or initiating directly from the [Pulumi Cloud Console](/docs/pulumi-cloud/deployments/reference/#pulumi-console).
+
+## Setup a new Pulumi Deployment
+
+First sign in to Pulumi Cloud via the [Pulumi sign in page](https://app.pulumi.com) or by running `pulumi login` via the Pulumi CLI.
+
+## Install and configure the GitHub App
+
+Next, install and configure the [Pulumi GitHub App](/docs/using-pulumi/continuous-delivery/github-app/) to enable Pulumi Deployment's New Project Wizard and push-to-deploy functionality. Follow the GitHub app [installation instructions](/docs/pulumi-cloud/deployments/reference/#github-app-installation) to ensure your Pulumi organization is set up to work seamlessly with Pulumi Deployments.
+
+## Create a New Project with the New Project Wizard
+
+To create a new Pulumi project, commit its code, and deploy it entirely from your browser, you will use the [New Project Wizard](/docs/pulumi-cloud/developer-portals/new-project-wizard/). The wizard allows you to start from one of Pulumi’s public templates, your [Organization Templates](/docs/pulumi-cloud/developer-portals/templates), or even generate a custom project using the cloud resources and your preferred language of choice with [Pulumi AI](link).
```

Comment:
We've mentioned the New Project Wizard several times at this point, but this is the first time we're linking to it. We should link to it earlier on.